crates/nu-command/tests/commands/redirection.rs

```diff
@@ -64,3 +64,72 @@ fn redirect_out() {
         assert!(output.out.contains("hello"));
     })
 }
+
+#[cfg(not(windows))]
```

Comment:
Do we need a test for the behavior/error behavior on Windows?

Reply:
Yeah I think so, going to investigate how to make the same stuff on Windows

Reply:
Just update it to run different script on different system